tests/e2e/testutils.go

```diff
@@ -323,6 +323,13 @@ func RunCmdOnNode(cmd string, nodename string) (string, error) {
 	return RunCommand(runcmd)
 }
 
+// RunCmdOnWindowsNode executes a command from within the given windows node
+func RunCmdOnWindowsNode(cmd string, nodename string) (string, error) {
+	communicator := "ssh"
+	runcmd := "vagrant " + communicator + " -c 'powershell.exe -Command \""+ cmd + "\"' " + nodename
+	return RunCommand(runcmd)
+}
+
```

Comment:
We currently override the default communicator for Windows VMs and set it to `winssh` at https://github.com/rancher/rke2/blob/master/tests/e2e/vagrantdefaults.rb#L9

Did you want to change the default there? Then we don't need this function.

Reply:
I was not aware of that, thanks!

The reason for this function is that I can't use the current [RunCmdOnNode](https://github.com/rancher/rke2/blob/master/tests/e2e/testutils.go#L319-L324) for Windows because of how picky powerhsell is with quotes. The command I need to run is:
vagrant winssh -c 'powershell.exe -Command "Restart-Service RemoteAccess"' windows-agent-0`. By creating this function I have the quotes correctly set and ready for any other command we might need to run in Windows in the future

Reply:
Okay, makes sense. But I don't think we need to define the communicator, since it will always be ssh
```suggestion
// RunCmdOnWindowsNode executes a command from within the given windows node
func RunCmdOnWindowsNode(cmd string, nodename string) (string, error) {
	runcmd := "vagrant ssh -c 'powershell.exe -Command \""+ cmd + "\"' " + nodename
	return RunCommand(runcmd)
}

```